### ISSUE TITLE
"Unsupported operation: add" for text component

### ISSUE THREAD

**Issue by mhaack** [adobe/da-universal#73]
Adding an image alt text or any other text field the first time, when the field was empty before, fails with "Unsupported operation: add".
Emptying a text field also fails with "Unsupported operation: add".

**Comment by mhaack** [comment]
Fixed in UE backend
